frontend/src/scenes/session-recordings/player/controller/Seekbar.tsx

```diff
@@ -80,7 +80,16 @@ export function Seekbar(): JSX.Element {
                             style={{ transform: `translateX(${thumbLeftPos}px)` }}
                         />
 
-                        <PlayerSeekbarPreview minMs={0} maxMs={sessionPlayerData.durationMs} seekBarRef={seekBarRef} />
+                        <PlayerSeekbarPreview
+                            minMs={0}
+                            maxMs={sessionPlayerData.durationMs}
+                            seekBarRef={seekBarRef}
+                            activeMs={
+                                sessionPlayerMetaData?.active_seconds
```

Comment:
annoyingly metadata activity is returned in seconds, converted them to ms here so that the preview component only receives one unit type

Reply:
Totally fine with this approach, but we could also get this as a selector from the segments as a more reliable value 🤷

Reply:
active_seconds is calculated from the segments on ingestion, and that's the value we're showing in the listing. So in theory it would be the same if calculated in a selector but I bet would sometime not be and we'd end up with a hard to discover bug 🤣